### DISCUSSION TITLE
Install apache 2.4.58

### DISCUSSION THREAD

**Discussion by quer0016** [JanssenProject/jans#7879] [Show and tell]
As is already known, in some operating systems Apache2 is automatically downloaded from their repo, but the version that normally downloads is not the latest one that solves some security problems since in the case of Ubuntu it installs version 2.4.52.
Before installing janssen

install the ppa repo: 
`sudo add-apt-repository ppa:ondrej/apache2 -y`
next:
```
sudo apt update
sudo apt install apache2
```
to check the version:
`apache2ctl -v`

It should show version 2.4.58

Janssen can now be installed can be seen from external connections with curl:
`curl -kI --insecure https://<host>`
If you have an SSL certificate, do not use --insecure

**Comment by ossdhaval** [comment]
Hey @quer0016 Many thanks for sharing this!! It'll be helpful to the community .. :partying_face:
